content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -3,7 +3,11 @@
 Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
 [remote storage](/doc/command-reference/remote).
 
-## Configure a Google Cloud project
+## Configure a Google Cloud project (Highly recommended optional step)
+
+We highly recommend to follow this guide and complete setup of your own Google
+Cloud project and
```

Comment:
```suggestion
Optionally but highly recommended, follow this guide to setup your own Google Cloud project and
```

Reply:
We should also probably expand on why this is beneficial to the user in another sentence in this same paragraph. E.g. he will not share rate limits with others, maybe more control/visibility for debugging, Idk, etc.

Reply:
yes, we def need to add motivation, mention that sharing potentially could lead for slower performance, give hints how to check if users hit limits, etc

Reply:
> give hints how to check if users hit limits

I'm not sure how to do this.

Could you guys help me summarize the benefits of crating your own GCP project?

I added some general info for now but it's probably not enough, please check it out.

Reply:
p.s. you know what I should just try all the gdrive config options again to help in QA which is probably needed anyway, I think that will allow me to review this and the whole doc anyway ⏳

Reply:
OK, added some precisions, including about this now.